exercises/all-your-base/src/test/java/ConverterTest.java

```diff
@@ -0,0 +1,232 @@
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class ConverterTest {
+
+    /*
+     * See https://github.com/junit-team/junit4/wiki/Rules for information on JUnit Rules in general and
+     * ExpectedExceptions in particular.
+     */
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+     @Test
+    public void singleBitOneToDecimal() {
+        final List<Integer> inputDigits = Collections.unmodifiableList(Arrays.asList(1));
+        final List<Integer> expected = Collections.unmodifiableList(Arrays.asList(1));
+        final List<Integer> actual = Converter.convert(inputDigits, 2, 10);
+        assertNotNull(actual);
+        assertFalse(actual.isEmpty());
+        assertEquals(expected, actual);
+    }
+
+    // @Test
```

Comment:
`@Ignore` is slightly but significantly different than commenting out `@Test` — that the user gets feedback that there's a skipped test.  This is a really important signal for practitioners so that they know they are not yet done when all the tests pass.

We've encouraged practitioners to use the command-line.  Running `gradle test` with tests commented out, it's easy to miss the fact that there are tests that are not yet addressed in their solution; it's far friendlier to see the "`SKIPPED`" output in the test run; that's why we're specifically using `@Ignore`, here.